src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,106 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 /**
  * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
  */
 public class ArrayList<T> implements List<T> {
+    private  int capasity = 7;
```

Comment:
@suleimanovayev Move 7 as a constant
```suggestion
    private static final int DEFAULT_CAPACITY=16;
    private  int capasity = DEFAULT_CAPACITY;
```